op-guide/configuration.md

```diff
@@ -14,249 +14,189 @@ The default TiDB ports are 4000 for client requests and 10080 for status report.
 - The TiDB server monitors this address.
-- The "0.0.0.0" monitors all network cards. If you have multiple network cards, specify the network card that provides service, such as 192.168.100.113.
-
-### `--join-concurrency int`
-
-- The number of goroutine when `join-concurrency` executes `join` concurrently
-- Default: 5
-- The number depends on the amount of data and data distribution, usually the larger the better, and a larger number means a larger CPU overhead.
+- The “0.0.0.0” monitors all network cards by default. If you have multiple network cards, specify the network card that provides service, such as 192.168.100.113.
 
 ### `-L`
 
-+ The log level
-+ Default: "info"
-+ You can choose from debug, info, warn, error, or fatal.
+- The log level
+- Default: "info"
+- You can choose from "debug", "info", "warn", "error", or "fatal".
 
-### `--lease`
+### `--log-file`
 
-+ The schema lease time in seconds
-+ Default: "10"
-+ This is the schema lease time that is used in online schema changes. The value will affect the DDL statement running time. Do not change it unless you understand the internal mechanism.
+- The log file
+- Default: ""
+- If this flag is not set, logs are written to "stderr". Otherwise, logs are stored in the log file which will be automatically rotated every day.
 
-### `--log-file`
+### `--log-slow-query`
 
-+ The log file
-+ Default: ""
-+ If this flag is not set, logs will be written to stderr. Otherwise, logs will be stored in the log file which will be automatically rotated every day.
+- The directory for the slow query log
+- Default: ""
+- If this flag is not set, logs are written to the file specified by `--log-file` by default.
 
 ### `--metrics-addr`
 
-+ The Prometheus pushgateway address
-+ Default: ""
-+ Leaving it empty stops the Prometheus client from pushing.
-+ The format is:
-    
+- The Prometheus Pushgateway address
+- Default: ""
+- Leaving it empty stops the Prometheus client from pushing.
+- The format is:
+
     ```
     --metrics-addr=192.168.100.115:9091
     ```
 
 ### `--metrics-interval`
 
-+ The Prometheus client push interval in seconds
-+ Default: 0
-+ Setting the value to 0 stops the Prometheus client from pushing.
+- The Prometheus client push interval in seconds
+- Default: 15s
+- Setting the value to 0 stops the Prometheus client from pushing.
 
-### `-P`
+### `-p`
 
-+ The monitoring port of TiDB services
-+ Default: "4000"
-+ The TiDB server accepts MySQL client requests from this port.
+- The monitoring port of TiDB services
+- Default: "4000"
+- The TiDB server accepts MySQL client requests from this port.
 
 ### `--path`
 
-- The path to the data directory for local storage engines like "goleveldb" and "BoltDB"
-- Do not set `--path` for the "memory" storage engine.
+- The path to the data directory for local storage engine like "mocktikv"
+- For `--store = tikv`, the path should be specified, for `--store = mocktikv`, if the path is not specified, the default value is used.
```

Comment:
(Grammar and wording)
-> "For `--store = tikv`,  you must specify the path; for `--store = mocktikv`, the default value is used if you do not specify the path"